windows/src/desktop/help/basic/toolbox_tasks/charactermap.md

```diff
@@ -312,4 +312,4 @@ Character Map.
 
 **Related Topics**
 
--   [???](#basic_toolbox)
+-   [Toolbox](toolbox)
```

Comment:
I think this is normally called `The Keyman Toolbox`

Reply:
k, I'll update all the Toolbox links